packages/core/src/stateUtils.ts

```diff
@@ -1638,8 +1638,7 @@ export function macrostep(
   }
 
   if (nextState.status !== 'active') {
-    // Perform the stop step to ensure that child actors are stopped
-    stopStep(nextEvent, nextState, actorCtx);
+    stopChildren(nextState, nextEvent, actorCtx);
```

Comment:
this likely tries to stop invoked actors for the second time and I have a feeling that this might be a problem as well, I'm gonna investigate this separately